apps/namehashlabs.org/components/1 - atoms/headline-banner.tsx

```diff
@@ -12,12 +12,12 @@ export const HeadlineBanner = () => {
       </div>
       <Link
         target="_blank"
-        href="https://twitter.com/NamehashLabs"
+        href="https://x.com/NamehashLabs"
         variant="underline"
         size="small"
       >
-        <p className="hidden sm:block">Follow us on Twitter</p>
-        <p className="block sm:hidden w-max">Follow us</p>
+        <span className="hidden sm:block">Follow us on Twitter</span>
+        <span className="block sm:hidden w-max">Follow us</span>
```

Comment:
When the screen width gets small enough, this "Follow Us" variation appears but it isn't getting the correct `<Link` UX.

Additionally, I don't know why, but the logic for the header isn't working correctly:

![CleanShot 2024-09-21 at 00 11 02](https://github.com/user-attachments/assets/dec68c55-b6ed-4b46-8c73-ba31491fd53b)

The hamburger menu is falling off the edge of the screen.

Reply:
Fixed